packages/theme-chalk/src/table.scss

```
@@ -235,6 +235,11 @@
     &.gutter {
       width: 0;
     }
+
+    &.cell {
```

```suggestion
    & .cell {
```

seems to bind the style to a wrong class name